src/main/java/com/cricket/project/dto/PlayerStatsDto.java

```diff
@@ -0,0 +1,12 @@
+package com.cricket.project.dto;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
```

Comment:
do we need this?

Reply:
no this file is not required anymore .. hence deleted